swap_meet/item.py

```diff
@@ -1,2 +1,30 @@
+from uuid import uuid1
+from datetime import datetime
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0, age=None):
+        self.id = int(uuid1()) if id is None else id
+        self.condition = condition
+        self.age = datetime.today().date() if age is None else datetime(*age).date()
+
+    def get_category(self):
+        return self.__class__.__name__
```

Comment:
Nice job writing this method in the super class and not repeating the logic in the sub classes 👍